src/bbc_meet_spotify/spotify.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
         self.spotify = spotipy.Spotify(auth=token)
         self.music_not_found = []
 
-    def add_songs_to_playlist(self, playlist_id: str, song_ids: List[str]):
+    def add_music_to_playlist(self, playlist_id: str, song_ids: List[str]) -> None:
```

Comment:
Since this is called by `add_albums()` and `add_songs()`, changed the function name to reflect what's happening